package.json

```diff
@@ -1,89 +1,96 @@
-    "npm": ">= 6.0.0"
-  },
-  "dependencies": {
-    "@financial-times/sass": "^1.31.0",
-    "@swc/cli": "^0.1.24",
-    "@swc/core": "^1.1.39",
-    "aliases": "^1.0.0",
-    "autoprefixer": "^9.7.6",
-    "bower": "^1.8.8",
-    "boxen": "^4.2.0",
-    "chalk": "^4.1.0",
-    "chokidar": "^3.4.0",
-    "colors": "^1.4.0",
-    "deglob": "^4.0.1",
-    "eslint": "^7.2.0",
-    "eslint-config-origami-component": "^2.0.1",
-    "eslint-plugin-import": "^2.21.1",
-    "execa": "^4.0.0",
-    "flush-write-stream": "^2.0.0",
-    "fs-extra": "^9.0.0",
-    "globby": "^11.0.0",
-    "is-ci": "^2.0.0",
-    "karma": "^5.1.0",
-    "karma-browserstack-launcher": "^1.5.2",
-    "karma-chrome-launcher": "^3.1.0",
-    "karma-mocha": "^2.0.0",
-    "karma-proclaim": "git+https://github.com/Financial-Times/karma-proclaim.git",
-    "karma-scss-preprocessor": "^4.0.0",
-    "karma-sinon": "^1.0.5",
-    "karma-sourcemap-loader": "^0.3.7",
-    "listr": "^0.14.3",
-    "listr-multiline-renderer": "^1.1.0",
-    "meow": "^7.0.1",
-    "merge-deep": "^3.0.2",
-    "mocha": "^8.0.1",
-    "mockery": "^2.1.0",
-    "mustache": "^4.0.1",
-    "node-sass": "^4.14.0",
-    "pa11y": "^5.3.0",
-    "parse-gitignore": "^1.0.1",
-    "portfinder": "^1.0.26",
-    "postcss": "^7.0.27",
-    "proclaim": "^3.6.0",
-    "root-check": "^1.0.0",
-    "sass-true": "^6.0.0",
-    "scrumple": "^1.2.1",
-    "serve-handler": "^6.1.2",
-    "sinon": "^9.0.2",
-    "snyk": "^1.319.0",
-    "stylelint": "^13.2.1",
-    "stylelint-config-origami-component": "^1.0.3",
-    "update-notifier": "^4.1.0"
-  },
-  "devDependencies": {
-    "@financial-times/origami-service-makefile": "^7.0.3",
-    "eslint-plugin-filenames": "^1.3.2",
-    "eslint-plugin-json": "^2.1.1",
-    "eslint-plugin-mocha": "^7.0.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-objects": "^1.1.1",
-    "eslint-plugin-promise": "^4.2.1",
-    "nixt": "^0.5.1",
-    "node-version": "^2.0.0",
-    "npm-prepublish": "^1.2.3",
-    "rimraf": "^3.0.2",
-    "unique-temp-dir": "^1.0.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Financial-Times/origami-build-tools.git"
-  },
-  "license": "MIT",
-  "scripts": {
-    "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect"
-  },
-  "snyk": true
+	"name": "origami-build-tools",
+	"version": "0.0.0",
+	"description": "Origami component development tools.",
+	"main": "./lib/origami-build-tools-cli.js",
+	"bin": {
+		"origami-build-tools": "./lib/origami-build-tools-cli.js",
+		"obt": "./lib/origami-build-tools-cli.js"
+	},
+	"engines": {
+		"node": ">= 12.0.0",
+		"npm": ">= 6.0.0"
+	},
+	"dependencies": {
+		"@financial-times/sass": "^1.31.0",
+		"@swc/cli": "^0.1.24",
+		"@swc/core": "^1.1.39",
+		"aliases": "^1.0.0",
+		"autoprefixer": "^9.7.6",
+		"bower": "^1.8.8",
+		"boxen": "^4.2.0",
+		"chalk": "^4.1.0",
+		"chokidar": "^3.4.0",
+		"colors": "^1.4.0",
+		"create-origami-component": "^0.2.13",
```

Comment:
It looks like this is a new dependency, I can't see it being used anywhere though. Is it being used?

Reply:
oh, weird. i guess we depend on it silently:

https://github.com/Financial-Times/origami-build-tools/blob/a1dd443e0fa0e9cf1fbd301eff91f0a6dd49c2d1/lib/tasks/init.js#L4-L6

we should probably lock that to a version? currently a change to `create-origami-component` can break working installs of `obt` and we wouldn't notice. maybe that's fine.

Reply:
oh! Nice work spotting that :-D Let's keep it defined in package.json, great idea 👍